package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "material-ui-json-schema-viewer",
+  "version": "1.0.0",
+  "main": "index.js",
+  "repository": "https://github.com/taskcluster/material-ui-json-schema-viewer",
```

Comment:
When we publish this package, we probably would want to just have the `src` and `build` directories published. We could add:

```
"files": [
  "build",
  "src"
],
```

Reply:
Updated changes!